maven-repository-metadata/src/main/mdo/metadata.mdo

```diff
@@ -338,10 +339,14 @@ under the License.
         </field>
       </fields>
     </class>
+
     <class java.clone="deep">
       <name>Plugin</name>
       <version>1.0.0+</version>
-      <description>Mapping information for a single plugin within this group</description>
+      <description>Mapping information for a single plugin within this group (deprecated).</description>
+      <annotations>
+        <annotation>@Deprecated</annotation>
```

Comment:
Who will actually perform the import for this annotation?

Reply:
"perform the import for this annotation"? what does that mean?
with this annotation, the class is marked deprecated (in generated java): I suppose nobody uses it apart the plugin-tools and Sonatype Nexus Staging
but what is useful is that it documents deprecation of data, given update of the data will disappear in the future with maven-compat removal

Reply:
I mean the Java import for the annotation class. Where is it? Will Modello itself generate the import statement?

Reply:
@Deprecated does not require import: I tested, generated java compiles, no problem